course/models.py

```diff
@@ -252,72 +301,97 @@ def __unicode__(self):
 # {{{ participation
 
 class ParticipationTag(models.Model):
-    course = models.ForeignKey(Course)
-
+    course = models.ForeignKey(Course,
+            verbose_name = 'Course ID')
     name = models.CharField(max_length=100, unique=True,
             help_text="Format is lower-case-with-hyphens. "
-            "Do not use spaces.")
+            "Do not use spaces.",
+            verbose_name = 'Name of ParticipationTag')
```

Comment:
`ParticipationTag` -> `participation tag` (no need to let the class name show through)